packages/validator/docs/actions.test.ts

```diff
@@ -1,234 +1,20 @@
+import * as data from './data/actions';
```

Comment:
Im not sure if im imagining it correctly, but I dont see a way of not making debugging very difficult. Currently if the test fails, it is clear which specification failed, but when just iterating through an array it might be difficult to find the specification that failed.

Reply:
I also don't see a nice way without listing the data in some custom format (like array of tripples <test_name, validator_input, validotor_output>) but that is unnecessary I guess.

Reply:
If the JSON objects were not in JS file, but instead were JSON files. You could "enforce" a folder structure (every fixture folder would have validator input, spec and output) and write a tests such that they would only need to folder name.